rust-examples/nocore-hello-world.rs

```diff
@@ -45,6 +45,9 @@ mod wasm {
     }
 }
 
+// This function will be run at Module start
+// e.g when using the interpreter
+#[wasm_start]
```

Comment:
Is the idea that the `wasm_start` function would eventually replace/supplement the `start` language item and call in to the usual Rust `main` function?